<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	background-color: #4a26fd;
	border-radius:5px;
	border: 1px solid  #4a26fd;
	font:  8pt &quot;FontAwesome&quot; ; 
	color: #ffffff

}

QPushButton:Hover{	
	background-color :#021aee
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbr_load_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Carga de imagen</string>
    </property>
    <widget class="QPushButton" name="btn_load_image">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>401</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar imagen</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_load_image">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_filters">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>421</width>
      <height>291</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>419</width>
       <height>289</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbr_add_filter">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Agregar filtros</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="btn_delimite">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delimitar </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_color">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_transformation">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Transf. libre</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_perspective">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Perspectiva</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Busqueda</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbr_end">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Guardar configuración</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="btn_save_json">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>401</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbr_info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>421</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Información</string>
    </property>
    <widget class="QLineEdit" name="txb_identifier">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="gbr_lbl_identifier">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Identificador único</string>
     </property>
    </widget>
    <widget class="QLabel" name="gbr_lbl_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txb_name">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="gbr_lbl_address">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dirección</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txb_address">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="gbr_lbl_token">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Token</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txb_token">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="gbr_lbl_model">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ruta modelo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txb_model">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbr_config">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuración</string>
    </property>
    <widget class="QLabel" name="gbr_lbl_video_source">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fuente de video</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txb_video_source">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_capture_image">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>401</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Capturar imagen</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
